src/app/public/modules/toast/toast.component.spec.ts

```diff
@@ -14,17 +14,17 @@ import {
   SkyToastTestComponent
 } from './fixtures';
 
-import {
-  SkyToastType
-} from './types';
-
 import {
   SkyToastComponent
 } from './toast.component';
 
 import {
   SkyToastService
 } from './toast.service';
+
+import {
+  SkyToastType
+} from './types';
 // #endregion
```

Comment:
While making these style fixes should we also remove the non-unnecessary regions?

Reply:
done

Reply:
+1 for the use of "non-unnecessary"

Reply:
Wow. I was on point with that one wasn't I.....